hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/crypto/AutoAccountCreationSuite.java

```diff
@@ -1038,6 +1046,86 @@ private HapiSpec hollowAccountCompletionWithCryptoTransfer() {
                                 }));
     }
 
+    private HapiSpec hollowAccountCompletionWhenHollowAccountSigRequiredInOtherReqSigs() {
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**question:** To be in the `otherReqSigs` means‏ that the hollow account is part of the transaction logic e.g. the sender in a crypto transfer, right?

Reply:
Correct!